p2p/discover/v5_udp_test.go

```diff
@@ -524,6 +524,13 @@ func TestUDPv5_lookup(t *testing.T) {
 	t.Parallel()
 	test := newUDPV5Test(t)
 
+	// Ensure lookupDistances includes 256 when necessary.
+	node255 := nodesAtDistance(test.table.self().ID(), 255, 1)[0]
```

Comment:
It's nice to have a test, but it should be in its own top-level test function.

Reply:
Good call, I agree. Updated 👍